content/docs/command-reference/exp/save.md

```diff
@@ -0,0 +1,163 @@
+# exp save
+
+Save the current project status as a [DVC experiment].
+
+[dvc experiment]: /doc/user-guide/experiment-management/experiments-overview
+
+## Synopsis
+
+```usage
+usage: dvc exp save [-h] [-q | -v] [-f]
+                   [--json] [-n <name>]
+                   [-I path]
+```
+
+## Description
+
+Provides a way to save the current status of your <abbr>project</abbr> as an
+<abbr>experiment</abbr> without polluting it with unnecessary commits, branches,
+directories, etc.
+
+Only files tracked by either Git or DVC are saved to the experiment. Use the
+`--include-untracked` (`-I`) option to explicitly include any untracked files in
+the experiment.
+
+`dvc exp save` creates <abbr>experiments</abbr> just like
+[`dvc exp run`](/doc/command-reference/exp/run), but does not require
+reproduction or setup of [pipeline stages](/doc/start/data-pipelines), making it
+possible to quickly start tracking, [comparing] and [persisting] experiments.
+
+[comparing]: /doc/user-guide/experiment-management/comparing-experiments
+[persisting]: /doc/user-guide/experiment-management/persisting-experiments
+
+## Options
+
+- `-n <name>`, `--name <name>` - specify a [unique name] for this experiment. A
+  default one will be generated otherwise, such as `exp-f80g4` (based on the
+  experiment's hash).
+
+- `-I <path>`, `--include-untracked <path>` - specify untracked file(s) to be
+  included in the saved experiment. Multiple files can be specified.
+
+- `-f`, `--force` - overwrite the experiment if an experiment with the same name
+  already exists.
```

Comment:
@dtrifiro The description in this PR is different from the one in the help text:

```
Save even if hash value for dependencies/outputs changed.
```

It seems like we are mixing two different force flags (`exp run -f` and `commit -f`) that do different things.

I don't think `--force` should be needed to commit changes to dvc-tracked files in `dvc exp save`, since I expect the command to behave basically like `dvc commit && git commit` (and similar to`dvc exp run`, which also does a forced `dvc commit`).  I opened https://github.com/iterative/dvc/pull/8691 to drop this check and make `--force` consistent with the description in this PR.

Reply:
I see we're now waiting on https://github.com/iterative/dvc/pull/8690 to clarify on this, is that right?

For now I'd go with the text from [commit -f](https://dvc.org/doc/command-reference/commit#-f):

> save the experiment even if hash values for dependencies or outputs did not change (same as `dvc commit -f`).